templates/epic.xml.jinja2

```diff
@@ -139,14 +139,36 @@
   {% endfor -%}
 {% endif -%}
 
-{% if 'hcal' in features %}
+
+{% if features is not defined or 'hcal' in features %}
   <documentation level="10">
-    ## hadronic calorimetry
+    ## Forward H calorimetry
   </documentation>
+  {% if features is not defined or features['hcal'] is none or 'forward_simplehcalwInlay' in features['hcal'] %}
+  <include ref="${DETECTOR_PATH}/compact/hcal/forward.xml"/>
+  <include ref="${DETECTOR_PATH}/compact/hcal/forward_insert.xml"/>
+  {% elif 'forward_simplehcal' in features['hcal'] -%}
+  <include ref="${DETECTOR_PATH}/compact/hcal/forward.xml"/>
+  {% elif 'lfhcal' in features['hcal'] -%}
+  <include ref="${DETECTOR_PATH}/compact/hcal/lfhcal.xml"/>
+
+  {% elif 'lfhcal_wInsert' in features['hcal'] -%}
+  <include ref="${DETECTOR_PATH}/compact/hcal/lfhcal_wInsert.xml"/>
+  <include ref="${DETECTOR_PATH}/compact/hcal/forward_insert.xml"/>
+  {% elif 'forward_insert' in features['hcal'] -%}
+  <include ref="${DETECTOR_PATH}/compact/hcal/forward_insert.xml"/>
+  {% endif -%}
+
+   ## Backward H calorimetry
+  {% if features is not defined or features['hcal'] is none or 'backward' in features['hcal'] %}
+  <include ref="${DETECTOR_PATH}/compact/hcal/backward.xml"/>
+  {% endif -%}
+
+    ## Central H calorimetry
+  {% if features is not defined or features['hcal'] is none or 'barrel_gdml' in features['hcal'] %}
+  <include ref="${DETECTOR_PATH}/compact/hcal/barrel_gdml.xml"/>
+  {% endif -%}
 
-  {% for hcal in features['hcal'] -%}
-  <include ref="${DETECTOR_PATH}/compact/hcal/{{ hcal }}.xml"/>
-  {% endfor -%}
 {% endif -%}
```

Comment:
Please don't change this file. The relevant files for the forward hcal should be included by specifying their path and fil name in the configuration.